src/api/utils/filter_functions.py

```diff
@@ -322,15 +323,31 @@ def get_users_filter_params(params):
         query.append(search)
 
       communities = params.get("community", None)
-      
       if communities and is_community_admin:
-        users = CommunityAdminGroup.objects.filter(Q(community__name__in=communities)| Q(community__id__in=communities), members__is_community_admin=True).values_list('members', flat=True).distinct()
+        if isinstance(communities[0], int):
```

Comment:
let's check upstairs and see if communities is even a list and not an empty list.  youre doing communities[0] directly which is dangerous unless you asserted that its a list and non-empty, else we risk it blowing up